server/templates/pages/src/style-guide.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="referrer" content="origin">
+        <meta name="robots" content="noindex,nofollow">
+        <link rel="stylesheet" href="/styles/main.css">
+        <link rel="stylesheet" href="/styles/system-font-main.css">
+
+        <script src="/bower_components/fxa-checkbox/checkbox.js"></script>
+    </head>
+    <body>
+        <label class="fxa-checkbox" for="checkbox-1">
+          <input type="checkbox" id="checkbox-1" class="checkbox_test" checked>
+          <span class="checkbox_label">Checkbox 1</span>
+        </label>
+        <label class="fxa-checkbox" for="checkbox-2">
+          <input type="checkbox" id="checkbox-2" class="checkbox_test">
+          <span class="checkbox_label">Checkbox 2</span>
+        </label>
+        <br/>
+        <p class="fxa-radio">
+          <label>
+            <input type="radio" id="option-1" class="radio_button" name="radio_test" value="1" checked/>
```

Comment:
either remove the classes (if you are not using them) OR convert from `_` to `-` in the names